Tests/test_DataStore_populate.py

```diff
@@ -0,0 +1,51 @@
+import unittest
+import os
+
+from unittest import TestCase
+from Store.DataStoreModule import DataStore
+
+FILE_PATH = os.path.dirname(__file__)
+TEST_DATA_PATH = os.path.join(FILE_PATH, "data")
```

Comment:
could you name the folder "sample_data" please?  Just to clarify what kind of data it is.

Reply:
Folder renamed.

https://github.com/debrief/python-import-library/pull/15/commits/0851cea2e57277328fe717f333a95231e1774baf